test/e2e/form-validation.test.js

```diff
@@ -10,130 +10,266 @@ describe('vl-form-validation', async () => {
   });
 
   it('als gebruiker zie ik een foutmelding als een verplicht veld niet is ingevuld in een form dat validatie doet', async () => {
-    const form = await vlFormValidationPage.getForm(1);
+    const formNr = 1;
+    const form = await vlFormValidationPage.getForm(formNr);
     const input = await form.getInputField('voornaam');
     await assert.eventually.isTrue(input.isRequired());
     await assert.eventually.isFalse(input.hasError());
 
-    await form.submit();
+    await form.submit(formNr);
```

Comment:
Waarom moet ge met submit nog eens dat nummer mee geven?

Reply:
De submit zoals die nu geïmplementeerd was, nam altijd de eerste knop op 't scherm. Eigenlijk zou form beter geëncapsuleerd moeten worden denk ik.

Reply:
Dan heb je de oplossing denk ik ... :-)

Reply:
Allez, die encapsulatie is er in vl-form, maar daar ligt geen dependency op van hier. Ik vermoed omdat we dan circulaire dependencies kregen. 
Dus ik zou ofwel effe ne VlForm hier ook maken dan ofwel submitForm(nummer) op de pagina steken. Form.submit(nummer) is verwarrend.

Reply:
ja, ik denk dat we die niet gebruiken omwille van circulaire dependencies

Reply:
Het gebruik van vl-ui-form introduceert geen circulaire dependencies, dus kan perfect gebruikt worden. De reden waarom dat tot op vandaag nog niet voorzien was, is puur omdat alleen het eerste formulier gebruikt werd in de e2e testen.

Reply:
Prima zoals het nu opgelost is:
```JS
async _getSubmitButton() {
  return new VlButton(this.driver, `${this.selector} [is="vl-button"]`);
}
```

Een alternatief kon zijn:
```
async _getSubmitButton() {
  const element = this.findElement('[is="vl-button"]');
  return new VlButton(this.driver, element);
}
```